### PR TITLE
Fugue SQL portion of fillna

### PR DESCRIPTION
**Pull request by kvnkho**
The unit test works but I'm not 100 percent sure everything is right here.

### CODE REVIEW

**Comment by goodwanghan** [comment]
line 413 in _visitor was not tested

File: tests/fugue_sql/test_workflow_parse.py [hunk 1]
```diff
@@ -515,6 +515,17 @@ def test_drop():
...
 
+def test_fill():
+    dag = FugueWorkflow()
+    a = dag.df([[None, 1], [1, None]], "a:int, b:int")
+    b = a.fillna({"a": 99, "b": -99})
+    assert_eq(
+        """
+    a=create [[NULL, 1],[1, NULL]] schema a:int, b:int
+    FILL NULLS PARAMS a:99, b:-99 FROM a
+    """,
```
Comment (goodwanghan) [inline tests/fugue_sql/test_workflow_parse.py:526]: Perfect, can you also anonymous case?

```
create [[NULL, 1],[1, NULL]] schema a:int, b:int
FILL NULLS PARAMS a:99, b:-99
```
this should also equal to your line 520 521

**Comment by kvnkho** [comment]
test coverage completed after adding anonymous cases.

File: fugue_sql/_visitors.py [hunk 2]
```diff
@@ -526,6 +526,18 @@ def visitFugueDropnaTask(self, ctx: fp.FugueDropnaTaskContext):
             params["subset"] = data["cols"]
         return df.dropna(**params)
 
+    def visitFugueFillnaTask(self, ctx: fp.FugueFillnaTaskContext):
+        data = self.get_dict(ctx, "params", "df")
+        if "df" in data:
+            df = data["df"]
+        else:
+            df = self.last
+        print("SEEE OUTPUT")
```
Comment (goodwanghan) [inline fugue_sql/_visitors.py:535]: Can you remove print?

**Review by goodwanghan — APPROVED** [review]
Please remove print before merge

**Pull request merged**